cmd/scaffold.go

```diff
@@ -726,9 +726,10 @@ func (fnb *FlowNodeBuilder) initProfiler() {
 			func(r uint) error { _ = runtime.SetMutexProfileFraction(int(r)); return nil }),
 	).Msg("could not register profiler setting")
 
-	fnb.Component("profiler", func(node *NodeConfig) (module.ReadyDoneAware, error) {
+	// registering as a DependableComponent with no dependencies so that it's started immediately on startup
```

Comment:
```suggestion
	// registering as a DependableComponent with no dependencies so that it's started immediately on startup
	// without being blocked by other component's Ready()
```